app/views/advanced/_guided_search_fields.html.erb

```diff
@@ -1,7 +1,8 @@
 
 <div class="advanced-search-field">
   <div class="search-field">
-    <%= select_tag('f1', options_for_select(advanced_key_value, guided_field(:f1, 'all_fields')), :class=>"search_field", 'aria-label': 'Options for advanced search') %>
+    <label for="f1" class="sr-only">Options for advanced search</label>
+    <%= select_tag('f1', options_for_select(advanced_key_value, guided_field(:f1, 'all_fields')), :class=>"search_field") %>
     <div name="row1">
       <%= text_field_tag "q1", label_tag_default_for(:q1), :class => 'form-control', autocorrect: "off", autocapitalize: "off", spellcheck: "false", 'aria-label': 'Advanced search terms' %>
```

Comment:
@christinach Can you also add a label element for these input fields, i.e., `q1`, `q2`, `q3`? Thank you.

Reply:
I will @axamei. Thank you!